auditlog/models/http_request.py

```diff
@@ -2,6 +2,7 @@
 # © 2015 ABF OSIELL <http://osiell.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
+from psycopg2.extensions import AsIs
```

Comment:
In the guidelines recommend us run `isort`
This lint detect this case and auto-fix.
But it is optional.
If you don't want is goos to us